audio-worklet/src/bitcrusher-demo.js

```diff
@@ -36,17 +33,17 @@ class BitcrusherDemo {
     this.masterGain_.connect(this.context_.destination);
 
     this.initializeGUI_(containerId);
-    this.loadSong_('audio/revenge.mp3', (song) => {
+    this.loadSong_('audio/revenge.mp3').then((song) => {
       this.songBuffer = song;
-      this.sourceButton_.enable()
+      this.sourceButton_.enable();
     });
   }
 
   async loadSong_(url, loadCompletedCallback) {
     const response = await fetch(url);
     const song = await response.arrayBuffer();
     const buffer = await this.context_.decodeAudioData(song);
-    loadCompletedCallback(buffer);
+    return buffer;
```

Comment:
Since we're not acting on the buffer in any other way than returning it, the way you propose works. So I will change to that.